models/event_api_handler.py

```diff
@@ -29,6 +29,8 @@ def get_list_of_events_from_query(cls):
                                  ).json()
         if apiResponse is None:
```

Comment:
Now we have 2 if statements, this should probably be pulled out into its own Verify response module that raises the exceptions for us if need be. The method get_list_of_events_from_query is now doing 2 things. Getting and validating. It should only be doing one (getting)